src/commands/Fun/Random/roll.ts

```diff
@@ -27,10 +50,11 @@ export default class extends SteveCommand {
 		super(store, file, directory, {
 			aliases: ['dice'],
 			description: 'Roll dice!',
-			examples: ['roll 1d6', 'roll d20', 'roll 5d10!', 'roll 1d8|4d6', 'roll 6d12k1', 'roll 6d12kl2'],
-			extendedHelp: oneLine`Use standard dice notation. You can roll up to 10 dice with up to 1,000 sides each.
+			examples: ['roll 1d6', 'roll d20', 'roll 4d6-5', 'roll 5d10!', 'roll 1d8|4d6', 'roll 6d12k1', 'roll 6d12kl2'],
+			extendedHelp: oneLine`Using standard dice notation: You can roll up to 10 dice with up to 1,000 sides each.
 				Add a \`!\` at the end of your roll to use exploding dice.
 				To keep the highest n, add \`k<n>\`; to keep the lowest n, add \`kl<n>\` (with n < amount of dice).
+				You can add a static positive or negative modifier to the sum of the roll using \`+<n>\` or \`-<n>\`.
 				You can do multiple rolls at once, separated by \`|\`.`,
 			usage: '<spec:dice> [...]',
 			helpUsage: '<number of dice>d<number of sides>'
```

Comment:
okay due to the big update recently, this is where the bulk of the necessary changes are gonna be. `helpUsage` and `examples` are unecessary now, as all of the command helps are in the `en-US` language file and everything has been internationalized. so i'd just port your additions to `examples` and `extendedHelp` over to that file.

Reply:
Fixed and passing checks in 2a9d846be6c76b21007c47ebb0851428269a7c3a

Reply:
Tested!